src/variable.h

```diff
@@ -254,6 +254,9 @@ class Variable {
     return m_tag >= std::tuple_size<detail::CoordDef::tags>::value +
                         std::tuple_size<detail::DataDef::tags>::value;
   }
+  bool isVariance() const {
```

Comment:
I think this should not be part of the `Variable` interface. The interface should be kept small and this does not seem like a fundamental concept. Client code can just as well use `var.tag() == Data.Variance`